README.md

```diff
@@ -1,31 +1,44 @@
 # Syne Tune
 
+[![Release](https://img.shields.io/badge/release-0.11-brightgreen.svg)](https://pypi.org/project/syne-tune/)
+[![Python Version](https://img.shields.io/badge/python-3.8-brightgreen.svg)](https://pypi.org/project/syne-tune/)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
 This package provides state-of-the-art distributed hyperparameter optimizers (HPO) where trials 
  can be evaluated with several backend options (local backend to evaluate them locally; SageMaker to evaluate them as 
  separate SageMaker training jobs; another backend with fast startup times is also in the making).
 
 ## Installing
 
-To install it "bare bones", you can do:
+To install Syne Tune from pip, you can simply do:
 
 ```bash
-pip install --upgrade pip
-git clone git@github.com:awslabs/syne-tune.git
-cd syne-tune
-pip install -e .[core]
+pip install syne-tune
 ```
 
-The `core` will allow you to run custom schedulers with different backends.
-
-If you want Ray Tune, our own Gaussian process based optimizers, or Bore optimizer, 
-you can run the same command replacing `core` by `raytune`, `gpsearchers`, `bore`, or several of them. You'll need:
+This will install a bare-bone version. If you want in addition to install our own Gaussian process based optimizers, Ray Tune or Bore optimizer, 
+you can run `pip install syne-tune[X]` where `X` can be 
 * `gpsearchers`: For built-in Gaussian process based optimizers
 * `raytune`: For Ray Tune optimizers
+* `benchmarks`: For installing all dependencies required to run all benchmarks
+* `extra`: For installing all the above
 * `bore`: For Bore optimizer
```

Comment:
is Bore included in extra as well? If so I would put 'extra' last

Reply:
No because one dependency does not work on the build environment (GPy).